### ISSUE TITLE
Show a description of what Bowtie is on the homepage

### ISSUE THREAD

**Issue by Julian** [bowtie-json-schema/bowtie#368]
It'd be nice to somewhere show some short description of what someone is looking at when visiting Bowtie's report.

**Comment by Min2who** [comment]
> It'd be nice to somewhere show some short description of what someone is looking at when visiting Bowtie's report.

can you explain a bit about what you want ?

cc @Julian

**Comment by Siddharth-Singh-2004** [comment]
@Julian , I am working on this issue. Kindly assign this issue to me.

**Comment by Siddharth-Singh-2004** [comment]
There are many ways of implementing this requirement:
1.  Leveraging tooltip (really short piece of text)
2. Showing text on hover (++tooltip with additional information)
3. Implementing cards (for more visual context, such as images )

As I see it, this action would be used only by somehow who is new to the UI and the first few times they use the UI until it all feels familiar. 

Here's an implementation going with option 2, to display text on hover:
Before the change:

<img width="1664" alt="Bowtie-Before" src="https://github.com/bowtie-json-schema/bowtie/assets/112953828/785c7581-a612-4370-9311-30be34bf9752">

After implementing hover for the docs button:
Clicking on Docs on the hover takes user to Bowtie documentation as before.

<img width="1664" alt="Bowtie-After-ThemeDark" src="https://github.com/bowtie-json-schema/bowtie/assets/112953828/9a2b303e-3f2b-4e99-bb88-f353352dd32c">

<img width="1663" alt="Bowtie-After-ThemeLight" src="https://github.com/bowtie-json-schema/bowtie/assets/112953828/dae641dc-67d3-4f72-90cf-bb40017b14f4">

**Comment by Julian** [comment]
Bowtie's docs are more about how you *use* Bowtie, particularly its CLI.

This issue is more about the report / the web page you're looking at.

If I asked you what you were looking at at https://bowtie.report -- what is it? What is it for?

That's the kind of information this issue is looking to add somewhere on the page.

**Comment by Siddharth-Singh-2004** [comment]
Just to clarify: Is it just on the navbar elements or any other section of what is visible? As a suggestion: Clicking the bowtie icon on the top left of the page would be the right place for that kind of info. However, we can consider adding such info to various "#" sections on the navbar as well. My implementation was to highlight the choice of UI elements and I could implement the same for any of the elements of the navbar.

**Comment by Julian** [comment]
This isn't about any of the navbar elements. It's to add detached information about what is in the report, and why someone may care!

**Comment by Siddharth-Singh-2004** [comment]
Got it!

**Comment by Siddharth-Singh-2004** [comment]
@Julian , Here's an implementation of what the UI could look like using Cards. If this is in the right direction, I can enhance what the text reads (with your help), change the styling of the text, #link appropriate sections of the report and add a few links to enrich the same

<img width="1221" alt="Bowtie card " src="https://github.com/bowtie-json-schema/bowtie/assets/112953828/42d7a149-598b-42b1-b5fb-bd7b26660589">

**Comment by Julian** [comment]
Two, maybe three sentences maximum is plenty I suspect -- there's a chance it would look better if we didn't have everything vertically stacked at this point but we could always tweak that afterwards.

**Comment by Siddharth-Singh-2004** [comment]
@Julian, Thank you for the quick revert. I will have a writeup complying with the suggestion above and run that by you. We certainly can do the UI differently. I'll come up with a few layout options and see which one works best given that analysts and developers like me would be using the UI. I would prefer more relevant data in an eyeball view, lesser clicks to get from one place to another.. basically, a data flow that will help me get to the bottom of the issue faster, so I can lower the turn around time.

**Comment by Siddharth-Singh-2004** [comment]
@Julian,  Here's what it looks like now: Other sections have been  included in screenshot to give a sense of relative size. The link for installation goes to this page... https://docs.bowtie.report/en/stable/... the page that I used to set up my bowtie environment. If you approve, I will go ahead with the pull request. 

<img width="1737" alt="Screenshot 2024-02-13 at 1 29 27 PM" src="https://github.com/bowtie-json-schema/bowtie/assets/112953828/fbd3b99d-4b98-4943-aa78-debb1ff15a3a">

**Comment by Julian** [comment]
Thanks but I again want to focus there on explaining what the page is in two sentences, so no need to talk about installation, nor to even mention Bowtie has a CLI.

If someone had no idea what they were looking at, what would you tell them? I doubt you'd start by explaining the 3 sections of the page.

**Comment by Siddharth-Singh-2004** [comment]
@Julian, , Sure. I tried answering these prompt in this issue thread #3 in which you clarify expectations: 
    What is Bowtie?
    What is the report showing?
    How can they install bowtie themselves
    Something very short perhaps that answers those each in two sentences and gets shown on the page would help!

Would this make the cut (will add relevant hyperlinks to text below):
Bowtie is a meta-validator of the JSON Schema specification, which coordinates executing other validator implementations, collecting and reporting on their results. To produce this report, Bowtie runs the official JSON Schema Test Suite, which are a set of tests meant to exercise compliance with the JSON Schema specifications.

**Comment by Julian** [comment]
I'd forgotten I'd mentioned "how can they install it"! Thanks for pointing that out.

I still say though that that's a lot less important than the first two there, so I was trying to focus you on getting those two right. We can decide whether to include the install link too.

> Bowtie is a meta-validator of the JSON Schema specification, which coordinates executing other validator implementations, collecting and reporting on their results. To produce this report, Bowtie runs the official JSON Schema Test Suite, which are a set of tests meant to exercise compliance with the JSON Schema specifications.

That's a lot closer! Maybe one more suggestion is that "meta-validator" is a term I invented because I think it makes sense to JSON Schema *implementers*, but I think JSON Schema *users* are less likely to know what it means, so maybe here's a suggestion:

> Bowtie is tool for understanding and comparing implementations of the JSON Schema specification, across all programming languages. The report below uses the official JSON Schema Test Suite to reveal and report any bugs or functionality gaps across implementations. You can also *use* Bowtie to get this information about your own schemas. For more information see its docs.

**Comment by Siddharth-Singh-2004** [comment]
@Julian, Taking inspiration from the above, I could craft the following:

Take #1:
Bowtie is a tool for assessing the compliance of a JSON schema implementation in a specific language by running it against the official JSON Schema Test Suite. The online report you are viewing is designed to help you compare various JSON schema implementations which are Bowtie’d, while highlighting areas which need fixing or enhancing. If you have a JSON schema you need assessed for compliance, Bowtie documentation has the details. 

Take #2:
Bowtie gives a single window view into compliance of JSON schema implementations in various programming languages by running it against the official JSON Schema Test Suite. Implementers can quickly get details of non-compliance and act upon them. If you have a schema implementation that needs to be assessed for compliance, our documentation has guidance to get Bowtie’d.  

Note: I have used Bowtie'd as a verb to amplify what bowtie really wants visitors to the site to do, get "tied" to bowtie.

**Comment by Siddharth-Singh-2004** [comment]
@Julian, Kindly let me know if you approve of any of the three takes above. Based on your comments, I can take it closer to your expectations.

**Comment by Julian** [comment]
What was your goal in rewriting the short sentences I gave you :)? Are you using AI to do these by any chance?

**Comment by Siddharth-Singh-2004** [comment]
@Julian,

Here's the ask from you as I understand from the exchanges above:
No more than 3 sentences and at max 4 lines of text (more than that is a little too much, I surmise).  I kept 2 stakeholders in mind as stated above. Added one sentence each to highlight the USP and a third one ,a common one, which may be of interest to both parties.

Absolutely no AI of any form. I don't think AI can do this very well given that bowtie is used in completely different context in common parlance. 

I searched the internet for all articles on Bowtie, mostly authored by you as blogs, and picked up bits and pieces of insights and added my own experience from the last few days to craft the messages. 

Ask from you:
--------------
Which of these takes are closer in terms of context? I can shorten it to make it easy on the eyes and improve assimilation for new visitors.
